modules/services/arrpc.nix

```diff
@@ -0,0 +1,31 @@
+{ config, pkgs, lib, ... }:
+let
+  inherit (lib) mkIf mkPackageOption mkEnableOption;
+
+  cfg = config.services.arrpc;
+in {
+  meta.maintainers = [ lib.maintainers.NotAShelf ];
+
+  options.services.arrpc = {
+    enable = mkEnableOption "arrpc";
+    package = mkPackageOption pkgs "arrpc" { };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.user.services.arRPC = {
+      Unit = {
+        Description =
+          "Discord Rich Presence for browsers, and some custom clients";
+        After = [ "graphical-session.target" ];
```

Comment:
It's always nice to make the systemd target configurable (see swayidle & and friends for inspiration).

Reply:
Good idea, I'll take a look at that.